llvm/test/Transforms/InstCombine/select-from-load.ll

```diff
@@ -0,0 +1,33 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -passes=instcombine -S -o - %s | FileCheck %s
+; REQUIRES: amdgpu-registered-target
```

Comment:
Move to AMDGPu subdirectory instead

Reply:
Although maybe this doesn't require the target, maybe just the datalayout with A5?

Reply:
Well, it still needs target. I tried only with data layout and it doesn't reproduce the crash.